# German translations for vigilo-connector.
# Copyright (C) 2010 ORGANIZATION
# This file is distributed under the same license as the vigilo-connector
# project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2010.
#
msgid ""
msgstr ""
"Project-Id-Version: vigilo-connector 2.0.0\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2010-11-19 12:16+0100\n"
"PO-Revision-Date: 2017-08-24 12:13+0200\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: de <LL@li.org>\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.4\n"

#: src/vigilo/connector/amqp.py:88
#, python-format
msgid "Channel open failed: %s"
msgstr ""

#: src/vigilo/connector/amqp.py:92
#, python-format
msgid "Error getting channel: %s"
msgstr ""

#: src/vigilo/connector/amqp.py:96
#, python-format
msgid "AMQP authentication failed: %s"
msgstr ""

#: src/vigilo/connector/amqp.py:100
#, python-format
msgid "Channel %(id)d was unexpectedly closed, disconnecting. Reason: %(reason)s"
msgstr ""

#: src/vigilo/connector/client.py:134
#, python-format
msgid "Server %(oldserver)s did not answer after %(attempts)d attempts"
msgstr ""

#: src/vigilo/connector/client.py:335
#, python-format
msgid "Lost connection to the bus (%s)"
msgstr ""

#: src/vigilo/connector/client.py:425
#, python-format
msgid "Sending failed: %(reason)s"
msgstr ""

#: src/vigilo/connector/client.py:446 src/vigilo/connector/client.py:652
#, python-format
msgid "Invalid configuration value for option \"%(key)s\"."
msgstr ""

#: src/vigilo/connector/client.py:451 src/vigilo/connector/client.py:657
#, python-format
msgid "Missing configuration value for option \"%(key)s\"."
msgstr ""

#: src/vigilo/connector/client.py:526
#, python-format
msgid "Creating lock file in %s"
msgstr ""

#: src/vigilo/connector/client.py:529
#, python-format
msgid "Lock file successfully created in %s"
msgstr ""

#: src/vigilo/connector/client.py:534
msgid "Error: lockfile found, another instance may be running."
msgstr ""

#: src/vigilo/connector/client.py:547
#, python-format
msgid "Error %(code)s: %(message)s"
msgstr ""

#: src/vigilo/connector/client.py:554
#, python-format
msgid "Error: %s"
msgstr ""

#: src/vigilo/connector/client.py:560
msgid "Timeout"
msgstr ""

#: src/vigilo/connector/client.py:619
msgid "No handler registered for this one-shot client"
msgstr ""

#: src/vigilo/connector/conffile.py:76
msgid "Received signal to reload the configuration"
msgstr ""

#: src/vigilo/connector/conffile.py:89
msgid "No configuration yet!"
msgstr ""

#: src/vigilo/connector/configure.py:57
#, python-format
msgid "Creating queue %s"
msgstr ""

#: src/vigilo/connector/configure.py:63
#, python-format
msgid "Deleting queue %s"
msgstr ""

#: src/vigilo/connector/configure.py:68
#, python-format
msgid "Creating exchange %(exchange)s of type %(type)s"
msgstr ""

#: src/vigilo/connector/configure.py:75
#, python-format
msgid "Deleting exchange %s"
msgstr ""

#: src/vigilo/connector/configure.py:83
#, python-format
msgid "Subscribing queue %(queue)s to exchange %(exchange)s (key: %(key)s)"
msgstr ""

#: src/vigilo/connector/configure.py:94
#, python-format
msgid "Unsubscribing queue %(queue)s from exchange %(exchange)s (key: %(key)s)"
msgstr ""

#: src/vigilo/connector/configure.py:106 src/vigilo/connector/configure.py:107
#, python-format
msgid "Can't find file '%s'"
msgstr ""

#: src/vigilo/connector/configure.py:116
#, python-format
msgid "Exchange %(name)s (%(type)s)"
msgstr ""

#: src/vigilo/connector/configure.py:127
#, python-format
msgid "Queue %s"
msgstr ""

#: src/vigilo/connector/configure.py:130
#, python-format
msgid "Queue %(queue)s subscribed to exchange %(exchange)s (key: %(key)s)"
msgstr ""

#: src/vigilo/connector/configure.py:145
msgid "Server address"
msgstr ""

#: src/vigilo/connector/configure.py:147
msgid "Bus user"
msgstr ""

#: src/vigilo/connector/configure.py:149
msgid "Bus password"
msgstr ""

#: src/vigilo/connector/configure.py:151
msgid "Show debugging information."
msgstr ""

#: src/vigilo/connector/configure.py:156
msgid "Vigilo bus configurator"
msgstr ""

#: src/vigilo/connector/configure.py:157
msgid "Subcommands"
msgstr ""

#: src/vigilo/connector/configure.py:163
msgid "Create an AMQP queue on the server."
msgstr ""

#: src/vigilo/connector/configure.py:165 src/vigilo/connector/configure.py:173
#: src/vigilo/connector/configure.py:199 src/vigilo/connector/configure.py:209
msgid "Queue name"
msgstr ""

#: src/vigilo/connector/configure.py:171
msgid "Delete an AMQP queue on the server."
msgstr ""

#: src/vigilo/connector/configure.py:179
msgid "Create an AMQP exchange on the server."
msgstr ""

#: src/vigilo/connector/configure.py:182
msgid "Exchange type"
msgstr ""

#: src/vigilo/connector/configure.py:183 src/vigilo/connector/configure.py:191
#: src/vigilo/connector/configure.py:200 src/vigilo/connector/configure.py:210
msgid "Exchange name"
msgstr ""

#: src/vigilo/connector/configure.py:189
msgid "Delete an AMQP exchange on the server."
msgstr ""

#: src/vigilo/connector/configure.py:197
msgid "Subscribe a queue to an exchange."
msgstr ""

#: src/vigilo/connector/configure.py:201 src/vigilo/connector/configure.py:211
msgid "Routing key"
msgstr ""

#: src/vigilo/connector/configure.py:207
msgid "Unsubscribe a queue from an exchange."
msgstr ""

#: src/vigilo/connector/configure.py:217
msgid "Reads the configuration from an INI file."
msgstr ""

#: src/vigilo/connector/configure.py:219
msgid "Configuration file"
msgstr ""

#: src/vigilo/connector/handlers.py:132
#, python-format
msgid "Could not initialize the queue: %(reason)s"
msgstr ""

#: src/vigilo/connector/handlers.py:269
msgid "Can't resume producing: not connected yet"
msgstr ""

#: src/vigilo/connector/handlers.py:354 src/vigilo/connector/handlers.py:388
#, python-format
msgid "Received message is not JSON-encoded: %r"
msgstr ""

#: src/vigilo/connector/handlers.py:524
msgid "Wrong values for parameters queue_batch_size and prefetch_count"
msgstr ""

#: src/vigilo/connector/handlers.py:900
#, python-format
msgid "Requeuing message (%(reason)s)."
msgstr ""

#: src/vigilo/connector/handlers.py:910
#, python-format
msgid "Error trying to save a message to the backup database: %s"
msgstr ""

#: src/vigilo/connector/handlers.py:972
#, python-format
msgid ""
"Can't understand the max_queue_size option, it should be an integer (or 0"
" for no limit). Current value: %s"
msgstr ""

#: src/vigilo/connector/handlers.py:980
msgid "The max_queue_size option is only available on Python >= 2.6. Ignoring."
msgstr ""

#: src/vigilo/connector/handlers.py:988 src/vigilo/connector/socket.py:119
#, python-format
msgid "Directory not found: '%(dir)s'"
msgstr "Verzeichnis: '%(dir)s' nicht gefunden"

#: src/vigilo/connector/handlers.py:993 src/vigilo/connector/socket.py:126
#, python-format
msgid "Wrong permissions on directory: '%(dir)s'"
msgstr ""

#: src/vigilo/connector/options.py:27
msgid "Load this settings.ini file"
msgstr ""

#: src/vigilo/connector/options.py:28
msgid "Unique ID for this instance"
msgstr ""

#: src/vigilo/connector/options.py:53
msgid "The configuration file does not exist"
msgstr ""

#: src/vigilo/connector/options.py:137 src/vigilo/connector/options.py:143
#: src/vigilo/connector/options.py:149
#, python-format
msgid "Can't parse publication value (%(type)s): \"%(value)s\""
msgstr ""

#: src/vigilo/connector/serialize.py:36
#, python-format
msgid "Got malformed message: %s"
msgstr "Fehlerhafte Nachricht: %s vorhanden"

#: src/vigilo/connector/serialize.py:47 src/vigilo/connector/serialize.py:96
#, python-format
msgid "Unknown/malformed message type: '%s'"
msgstr "Unbekannter/fehlerhafter Nachrichtentyp: '%s'"

#: src/vigilo/connector/socket.py:40
#, python-format
msgid "Unparsable line: %s"
msgstr ""

#: src/vigilo/connector/ssl.py:102
#, python-format
msgid ""
"Machine hostname (%(hostname)s) does not match certificate CommonName "
"(%(cn)s)"
msgstr ""

#: src/vigilo/connector/status.py:65
msgid "OK: running"
msgstr ""

#: src/vigilo/connector/status.py:173
#, python-format
msgid "Stats for %(service)s: %(name)s = %(value)s"
msgstr ""

#: src/vigilo/connector/status.py:217
#, python-format
msgid "Invalid configuration option for publications: (%(error)s)."
msgstr ""

#: src/vigilo/connector/store.py:190 src/vigilo/connector/store.py:290
#, python-format
msgid "Could not fill the output buffer: %s"
msgstr ""

#: src/vigilo/connector/store.py:232
#, python-format
msgid "Could not VACUUM the database: %s"
msgstr ""

